package/MDAnalysis/analysis/density.py

```diff
@@ -550,6 +589,14 @@ def density_from_Universe(universe, delta=1.0, atomselection='name OH2',
            Show status update every `interval` frame [1]
     parameters : dict (optional)
             `dict` with some special parameters for :class:`Density` (see docs)
+    gridcenter : np.array float (optional)
+            User defined grid box center (x,y,z: numpy array float 32) in Angstroem [None]
+    xdim : float (optional - ignored if gridcenter is None)
```

Comment:
Please follow the doc guidelines, namely using numpy style. I don't think that allows for extensive text in parentheses. Can you please rewrite this (and others) as
```rest
xdim : float (optional)
           User defined x dimension box edge in ångström; ignored if gridcenter is ``None`` [2.0]
```